apps/prairielearn/src/migrations/20240131102102_group_roles__can_assign_roles__backfill.sql

```diff
@@ -0,0 +1,4 @@
+-- This is done with a regular migration instead of a batched migration because the number of rows in production is small.
+UPDATE group_roles
+SET
+  can_assign_roles = can_assign_roles_at_start;
```

Comment:
Double-checking: when we introduced `can_assign_roles`, we ensured that for any newly written or updated rows, that these would have the same value, correct? This is only to handle any hypothetical rows for courses which haven't been synced since the introduction of `can_assign_roles`?

Reply:
Yes, that's correct. I don't expect to see any cases of these laying around (possibly example/test course), this is extra caution.